pom.xml

```diff
@@ -330,6 +330,11 @@
             <activation>
                <jdk>11</jdk>
             </activation>
+            <properties>
+                <java.version.required>11</java.version.required>
+                <javac.source.version>11</javac.source.version>
+                <javac.target.version>11</javac.target.version>
+            </properties>
```

Comment:
Maybe this should go to a separate PR.

Reply:
maybe, we will see what (if) they say anything, without this it does not compile

Reply:
you are right, somehow now it builds without, i remove it